app/i18n/cz/feedback.php

```diff
@@ -84,7 +84,7 @@
 			'cache_cleared' => '<em>%s</em> cache has been cleared',	// TODO - Translation
 			'deleted' => 'Kanál byl smazán',
 			'error' => 'Kanál nelze aktualizovat',
-			'internal_problem' => 'RSS kanál nelze přidat. Pro detaily <a href="%s">zkontrolujte logy FreshRSS</a>.',
+			'internal_problem' => 'RSS kanál nelze přidat. Pro detaily <a href="%s">zkontrolujte logy FreshRSS</a>. You can try force adding by appending <code>#force_feed</code> to the URL.',	// TODO - Translation
```

Comment:
I don't think this will quite work with the automated tools unfortunately, but the CI will say soon enough.

Reply:
fixed it